pallets/crowdloan-claim/src/lib.rs

```diff
@@ -320,14 +344,48 @@ pub mod pallet {
     #[pallet::storage]
     #[pallet::getter(fn processed_claims)]
     pub(super) type ProcessedClaims<T: Config> =
-        StorageMap<_, Blake2_128Concat, T::RelayChainAccountId, T::BlockNumber>;
+        StorageMap<_, Blake2_128Concat, (T::RelayChainAccountId, Index), bool>;
+
+    #[pallet::type_value]
+    pub fn OnIndexEmpty() -> Index {
+        0
+    }
+
+    #[pallet::storage]
+    #[pallet::getter(fn curr_index)]
+    pub type CurrIndex<T: Config> = StorageValue<_, Index, ValueQuery, OnIndexEmpty>;
```

Comment:
is it not possible to give the default here like pallet V1 ?

Reply:
To be honest, I don't know. Thought, that this replaces the default, as you can also put arbitrary logic in the function like with the build closure.

Reply:
hmm. okay. That one seems much simple.